base/libc.jl

```diff
@@ -176,13 +176,20 @@ struct TimeVal
    usec::Int64
 end
 
-function TimeVal()
-    tv = Ref{TimeVal}()
-    status = ccall(:jl_gettimeofday, Cint, (Ref{TimeVal},), tv)
+struct TimeSpec
+   sec::Int64
+   nsec::Int64
```

Comment:
```suggestion
   nsec::Int32
```

This is actually a Clong, which is sometimes Int32 and sometimes Int64, and sometimes just plain inconvenient to know what the platform will do. But if we use libuv, then it is explicitly Int32.